src/main/java/at/ac/fhcampuswien/snake/service/HighscoreService.java

```diff
@@ -0,0 +1,147 @@
+package at.ac.fhcampuswien.snake.service;
+
+import at.ac.fhcampuswien.snake.util.Player;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static at.ac.fhcampuswien.snake.util.Constants.HIGHSCORE_SEPARATOR;
+
+/**
+ * Utility Class to read from and save to a text file that stores the high scores.
+ */
+public class HighscoreService {
+
+    private final static Logger LOG = LoggerFactory.getLogger(HighscoreService.class);
+
+    /**
+     * This method looks for an existing high score file on disk. If it does not exist - it creates one.
+     *
+     * @return file object that points to the file on disk.
+     * @throws IOException if the program can not read from file / process gets interrupted.
+     */
+    private static File getHighscoresFile() throws IOException {
+        String path = "src/main/resources/highscores.txt";
+        File highscoreFile = new File(path);
+        highscoreFile.createNewFile();
```

Comment:
Result of 'File.createNewFile()' is ignored

Reply:
Will check this in future release. Stakeholder is waiting for this patch.